### PR TITLE
fix(platform-browser): mark Meta and Title services as tree shakable …

### PR DESCRIPTION
**Pull request by vikerman**
…providers

This lets services that use Meta and Title services to be tree shakable and provided in root.

### CODE REVIEW

File: packages/platform-browser/src/browser/title.ts [hunk 1]
```diff
@@ -22,7 +22,7 @@ import {DOCUMENT} from '../dom/dom_tokens';
  *
  * @experimental
  */
-@Injectable()
+@Injectable({providedIn: 'root'})
```
Comment (alxhub) [inline packages/platform-browser/src/browser/title.ts]: Presumably this is provided by a module (`BROWSER_MODULE_PROVIDERS` in `browser.ts`). Shouldn't it be removed from there?

**Comment by alxhub** [comment]
I only see the `Title` service being updated in this PR, not `Meta`.

**Review by alxhub — CHANGES_REQUESTED** [review]

File: packages/platform-browser/src/browser/title.ts (hunk 1 above)
Comment (vikerman) [reply packages/platform-browser/src/browser/title.ts]: Done

**Review by alxhub — APPROVED** [review]

**Review by mhevery — APPROVED** [review]

**Comment by gregmagolan** [comment]
Not sure how, but this broke the travis CI non-bazel build.

`Rollup platform-browser` dies. Error is: 

```/Users/greg/google/gregmagolan/angular/node_modules/.bin/rollup -i /Users/greg/google/gregmagolan/angular/dist/packages/platform-browser/platform-browser.js -o /Users/greg/google/gregmagolan/angular/dist/packages-dist/platform-browser/fesm2015/platform-browser.js --sourcemap -f es --banner BANNER_TEXT

/Users/greg/google/gregmagolan/angular/dist/packages/platform-browser/platform-browser.js → dist/packages-dist/platform-browser/fesm2015/platform-browser.js...
[!] Error: Could not resolve '../../../../dist/packages/common' from dist/packages/platform-browser/src/browser/meta.js
Error: Could not resolve '../../../../dist/packages/common' from dist/packages/platform-browser/src/browser/meta.js
    at error (/Users/greg/google/gregmagolan/angular/node_modules/rollup/dist/rollup.js:181:12)
    at /Users/greg/google/gregmagolan/angular/node_modules/rollup/dist/rollup.js:10121:7
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
    at Function.Module.runMain (module.js:695:11)
    at startup (bootstrap_node.js:188:16)
    at bootstrap_node.js:609:3
```

**Pull request closed without merging**